repos/system_upgrade/el7toel8/actors/selinux/selinuxapplycustom/actor.py

```diff
@@ -0,0 +1,115 @@
+import os
+from shutil import rmtree
+
+from leapp.actors import Actor
+from leapp.models import SELinuxModules, SELinuxCustom, SELinuxRequestRPMs, CheckResult
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+from leapp.libraries.stdlib import run, CalledProcessError
+
+WORKING_DIRECTORY = '/tmp/selinux/'
+
+class SELinuxApplyCustom(Actor):
+    '''
+    Re-apply SELinux customizations from RHEL-7 installation
+
+    Re-apply SELinux policy customizations (custom policy modules and changes
+    introduced by semanage). Any changes (due to incompatiblity with RHEL-8
+    SELinux policy) are reported to user.
+    '''
+    name = 'selinuxapplycustom'
+    consumes = (SELinuxCustom, SELinuxModules, )
+    produces = (CheckResult, )
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag, )
+
+    def process(self):
+        # cil module files need to be extracted to disk in order to be installed
+        try:
+            # clear working directory
+            rmtree(WORKING_DIRECTORY)
+        except OSError:
+            # expected
+            pass
+        try:
+            os.mkdir(WORKING_DIRECTORY)
+        except OSError:
+            self.log.info("Failed to access working directory! Aborting.")
```

Comment:
Failed to access or failed to create?

Reply:
Create... thank you, missed this after refactoring.

Reply:
Actually, you don't even know. 
```suggestion
        except OSError as e:
            self.log.info("Skipping to apply custom SELinux modules due to a failure while creating the working directory! Error reason: %s.", str(e))
```

I'd do something like this